handlers/services.go

```diff
@@ -43,6 +43,19 @@ func ServiceList(w http.ResponseWriter, r *http.Request) {
 	RespondWithJSON(w, http.StatusOK, serviceList)
 }
 
+// NamespaceMetrics is the API handler to fetch metrics to be displayed, related to all
+// services in the namespace
+func NamespaceMetrics(w http.ResponseWriter, r *http.Request) {
```

Comment:
Should rather go in `namespaces.go` , no? Since it's in the same package you will still have access to `getMetric`

Reply:
That makes sense. I'll change, thanks.